Zilon.Core/Zilon.Core/Commands/Sector/TacticCommandBase.cs

```diff
@@ -31,10 +31,10 @@ public void Execute()
 
             ExecuteTacticCommand();
 
-            if (UpdateGameLoop)
-            {
-                _gameLoop.Update();
-            }
+            //if (UpdateGameLoop)
+            //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)